src/components/PrivateRoute.jsx

```diff
@@ -1,19 +1,54 @@
-import React from 'react';
+import '@reshuffle/code-transform/macro';
```

Comment:
Really cool module. We should publish it as a package. (*not* in this PR, obviously!).